servicecontrol/monitoring-instances/installation/index.md

```diff
@@ -9,7 +9,7 @@ The ServiceControl installation package includes a utility to manage the install
 
 Use a Monitoring instance to vizualize [endpoint performance metrics](/monitoring/#endpoint-performance) in ServicePulse.
 
-Monitoring instances are not as [resource intensive as the primary and audit instances](/servicecontrol/servicecontrol-instances/hardware.md) and can be run on the same machine as the primary instance.
+Monitoring instances are not as [resource intensive as the error and audit instances](/servicecontrol/servicecontrol-instances/hardware.md) and can be run on the same machine as the Error instance.
```

Comment:
As much as I dislike capitalising these, it seems they should be for consistency?

```suggestion
Monitoring instances are not as [resource intensive as the Error and Audit instances](/servicecontrol/servicecontrol-instances/hardware.md) and can be run on the same machine as the Error instance.
```